src/Components/Publishing/Sections/ImageWrapper.tsx

```diff
@@ -55,10 +56,16 @@ export class ImageWrapper extends React.PureComponent<Props, any> {
   }
 
   render() {
-    const { layout, linked, index, ...blockImageProps }: any = this.props
+    const {
+      layout,
+      linked,
+      editing,
+      index,
+      ...blockImageProps,
```

Comment:
@damassi I'm having an issue where tslint keeps adding a trailing comma after `...blockImageProps`, but commas are not allowed for the spread. 

I tried adding `tslint:disable` to the top of the file, but it still gets added.  Any ideas of what rules I could look at to prevent this?

Reply:
Hmm, no clue... I would dig around online!

Reply:
Yeah is definitely coming from prettier, stops if I disable for this file.  Supposedly adding `--parser typescript` should stop adding it, but haven't managed to make it work.